src/yio-interface/entities/mediaplayerinterface.h

```diff
@@ -93,13 +105,25 @@ class MediaPlayerDef : public QObject {
         C_VOLUME_SET,
         C_VOLUME_UP,
         C_VOLUME_DOWN,
+        C_CURSOR_UP,
+        C_CURSOR_DOWN,
+        C_CURSOR_LEFT,
+        C_CURSOR_RIGHT,
+        C_CURSOR_OK,
+        C_MENU,
+        C_CHANNEL_UP,
+        C_CHANNEL_DOWN,
+        C_MUTE_TOGGLE,
         C_MUTE,
         C_BROWSE,
         C_SEARCH,
+        C_MEDIAPLAYERCOMMAND,
```

Comment:
What does the `C_MEDIAPLAYERCOMMAND` command do?
Send a command to media player or start the media player?
If it starts the media player, please rename to `C_MEDIAPLAYER`.

Reply:
Changed it to C_MEDIAPLAYERTEXTINPUT this is used if a textbox in kodi needs input like a search or password for netflix